tests/test_importer.py

```diff
@@ -30,7 +32,10 @@ def data_import(location_data_import, data_folder):
 
 @pytest.mark.django_db
 def test_no_sources_missing(data_import):
+    assert 'does not have sources' not in data_import.getvalue()
```

Comment:
We have a lot of ways of reporting the same error. These changes add additional log messages to check for. I'll open a separate issue to unify our logging language.